package.json

```diff
@@ -34,6 +34,7 @@
     "start:one-trust-notices": "turbo start --filter pagopa-interop-one-trust-notices",
     "start:datalake-data-export": "turbo start --filter pagopa-interop-datalake-data-export",
     "start:authorization-server": "turbo start --filter pagopa-interop-authorization-server",
+    "start:token-generation-readmodel-checker-verifier": "turbo start --filter pagopa-interop-token-generation-readmodel-checker-verifier",
```

Comment:
may we rename the package to `token-generation-readmodel-checker` 🙏 
(having both `checker` and `verifier` it's redundant)